src/neo/Network/P2P/TaskManager.cs

```diff
@@ -266,5 +285,13 @@ internal protected override bool IsHighPriority(object message)
                     return false;
             }
         }
+
+        internal protected override bool ShallDrop(object message, IEnumerable queue)
+        {
+            if (!(message is TaskManager.NewTasks tasks)) return false;
+            // Remove duplicate tasks
+            if (queue.OfType<TaskManager.NewTasks>().Any(x => x.Payload.Type == tasks.Payload.Type && x.Payload.Hashes.SequenceEqual(tasks.Payload.Hashes))) return true;
```

Comment:
This is the last question. I still believe we should remove this.
https://github.com/neo-project/neo/pull/899#discussion_r315172978

Reply:
This version can proceed the same `inv` after 1 min.

https://github.com/neo-project/neo/blob/75f4470356aa3ef97d5752932dd0214a80ad4953/src/neo/Network/P2P/ProtocolHandler.cs#L42

Reply:
Still there could be some attack from the node, which also tracks this `1min` interval and makes that for locking a node...aheuahuea
However, I think it is not worth considering this right now.

Reply:
@erikzhang @yongjiema